### ISSUE TITLE
Cannot create tray Error: Cannot find build dir

### ISSUE THREAD

**Issue by Alexandero89** [taw00/joplin-rpm#6]
Added the repository via dnf and installing the latest joplin package (version: 1.7.11-1.fc33.taw.x86_64).
Then started joplin, but it gets an exception and is not starting. 

OS: Fedora 33

Error:
```
Cannot create tray Error: Cannot find build dir
    at ElectronAppWrapper.buildDir (/usr/share/org.joplinapp.joplin/desktop/resources/app.asar/ElectronAppWrapper.js:232:23)
    at ElectronAppWrapper.createTray (/usr/share/org.joplinapp.joplin/desktop/resources/app.asar/ElectronAppWrapper.js:252:43)
    at electron/js2c/browser_init.js:213:7789
    at IpcMainImpl.<anonymous> (electron/js2c/browser_init.js:213:4246)
    at IpcMainImpl.emit (events.js:310:20)
    at WebContents.<anonymous> (electron/js2c/browser_init.js:157:8848)
    at WebContents.emit (events.js:310:20)
Cannot create tray Error: Cannot find build dir
    at ElectronAppWrapper.buildDir (/usr/share/org.joplinapp.joplin/desktop/resources/app.asar/ElectronAppWrapper.js:232:23)
    at ElectronAppWrapper.createTray (/usr/share/org.joplinapp.joplin/desktop/resources/app.asar/ElectronAppWrapper.js:252:43)
    at electron/js2c/browser_init.js:213:7789
    at IpcMainImpl.<anonymous> (electron/js2c/browser_init.js:213:4246)
    at IpcMainImpl.emit (events.js:310:20)
    at WebContents.<anonymous> (electron/js2c/browser_init.js:157:8848)
    at WebContents.emit (events.js:310:20)
```

Do you maybe know a solution?

**Comment by bivalence** [comment]
I ran into a similar stack also, although I only get it when trying to open the web clipper options panel.  I unpacked the app.asar file that comes in the rpm and checked out the ElectronAppWrapper.js file. The buildDir function that is failing is looking for a 'build' dir, and fails when it doesn't exist.  So I did:
`sudo mkdir /usr/share/org.joplinapp.joplin/desktop/resources/build`
and the options page for the web clipper service now displays.  Sounds like this could be a packaging problem, but I don't know.  This doesn't happen in Joplin deb packages.
